website/docs/r/sddc.html.markdown

```diff
@@ -73,7 +73,7 @@ The following arguments are supported:
 
 * `account_link_sddc_config` - (Optional) The account linking configuration object.
 
-* `host_instance_type` -  (Optional) The instance type for the esx hosts in the primary cluster of the SDDC. Possible values : I3_METAL, R5_METAL.
+* `host_instance_type` -  (Optional) The instance type for the esx hosts in the primary cluster of the SDDC. Possible values : I3_METAL, I3EN_METAL, R5_METAL. I3EN_METAL does not currently support single host (1NODE) or two host SDDC deployment.
```

Comment:
I don't think this is optional anymore.

Reply:
it has been defined as optional in the schema